bin/radical-pilot-hello.sh

```diff
@@ -82,8 +88,17 @@ printf "$PREFIX : RANK    : $MPI_RANK\n"
 printf "$PREFIX : THREADS : $THREAD_NUM\n"
 printf "$PREFIX : SLEEP   : $ARG\n"
 
-# if so requested, sleep for a bit
-sleep $ARG
+# check if `stress-ng` is installed
+export PATH="$PATH:$RADICAL_RESOURCE_SBOX/install/bin"
+STRESS=$(which stress-ng)
```

Comment:
may be to redirect error stream here (`STRESS=$(which stress-ng 2> /dev/null)`)?
to avoid unnecessary messages in `.err` file
```
$ cat task.000000.launch.out 
which: no stress-ng in (...)
```

Reply:
Thanks, this is added now.